Modules/IO/JPEG/test/itkJPEGImageIOTest.cxx

```diff
@@ -30,30 +30,25 @@ itkJPEGImageIOTest(int argc, char * argv[])
 
   if (argc < 3)
   {
-    std::cerr << "Usage: " << itkNameOfTestExecutableMacro(argv) << " Input Output\n";
+    std::cerr << "Missing parameters." << std::endl;
+    std::cerr << "Usage: " << itkNameOfTestExecutableMacro(argv);
+    std::cerr << " inputFilename outputFilename" << std::endl;
```

Comment:
This block is also questionable (everwhere). IMHO, simple comment about parameters is enough. They are hard-coded in _cmake_ file. By several thousands tests running many dozens times per day it is a lot of energy consumption to compile that useless blocks. IMHO. Not a request to change it now.